integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DbUtils.java

```diff
@@ -719,4 +689,331 @@ public static void updateRcuAccessSecret(String secretName, String namespace,
 
   }
 
+  /**
+   * Install Oracle Database Operator.
+   *
+   * @param namespace name of the namespace
+   * @throws IOException when fails to modify operator yaml file
+   */
+  public static void installDBOperator(String namespace) throws IOException {
+    String operatorYamlFile = DOWNLOAD_DIR + "/oracle-database-operator.yaml";
+    String certManager = "https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.yaml";
```

Comment:
Can we move this hard-coded URL to TestConstant class

Reply:
done